sky/provision/aws/instance.py

```diff
@@ -175,6 +186,20 @@ def cleanup_ports(
         'Values': [sg_name]
     }])
     if len(list(sgs)) != 1:
-        raise ValueError(f'Expected security group {sg_name} not found. '
-                         'Cannot cleanup ports.')
-    list(sgs)[0].delete()
+        logger.warning(f'Expected security group {sg_name} not found. '
+                       'Skip cleanup.')
```

Comment:
This logging seems inaccurate when `len(list(sgs)) > 1`?

Reply:
I just realized that the sg name can be the same... Now just skip cleanup since there is no vpc id in provider config and left a TODO